src/dev.html

```diff
@@ -18,6 +18,8 @@
 
     <header id="site-header"></header>
 
+    <mkt-nav role="navigation" id="site-nav"></mkt-nav>
```

Comment:
The inflexibility I see as problematic is that we're stuck to a rigid HTML template that can't be touched on a per-project basis. We can start injecting things all over the place with JS, but that gets messy pretty quickly.